tests/h/views/api_groups_test.py

```diff
@@ -8,28 +8,36 @@
 from pyramid.httpexceptions import HTTPNoContent, HTTPBadRequest
 
 from h.views import api_groups as views
+from h.services.profile_group import ProfileGroupService
+from h.services.group import GroupService
 
 
 @pytest.mark.usefixtures('profile_group_service')
 class TestGroups(object):
     def test_groups_proxies_to_service(self, pyramid_request, profile_group_service):
         views.groups(pyramid_request)
 
-        assert profile_group_service.called_once()
+        profile_group_service.all.assert_called_once()
 
     def test_groups_passes_authority_parameter(self, pyramid_request, profile_group_service):
         pyramid_request.params = {'authority': 'foo.com'}
 
         views.groups(pyramid_request)
 
-        assert profile_group_service.called_once_with(pyramid_request, 'foo.com')
+        profile_group_service.all.assert_called_once_with(user=mock.ANY,
+                                                          authority='foo.com',
```

Comment:
Doesn't this do the same as the `c_kwargs` check below?

Reply:
Yes, good catch. I had left it in place while I was "getting the other variant to work" and it inadvertently got left behind. I shall eliminate.